packages/reports/addon/components/filter-builders/base.js

```diff
@@ -27,42 +23,47 @@ export default Component.extend({
    * @property {String} filter.operator.valuesComponent - name of component to use for selecting filter values
    * @property {Array} filter.values
    */
-  filter: undefined,
+  get filter() {
```

Comment:
does this need to be a getter?

Reply:
I can try it with just `filter;`, but `filter = undefined` results in an error. I think it's trying to do `this.filter = undefined` during the constructor but in subclasses we have `get filter()` already defined, so it thinks we are trying to set a getter only property

Reply:
Ok coming back to this, if we do
```
filter = undefined; // breaks
filter; // breaks
// filter; // works, but confusing
get filter() { // works, slightly less confusing
  return undefined;
}
```

basically the same thing with `supportedOperators` so I'm opting to leave it as is